pkg/utils/release/controller.go

```diff
@@ -111,6 +139,20 @@ func (s *SuiteController) GetRelease(releaseName, releaseNamespace string) (*v1a
 	return release, err
 }
 
+// GetReleaseInNamespace returns the list of releases in the given namespace.
+func (s *SuiteController) GetReleaseInNamespace(namespace string) (*v1alpha1.Release, error) {
```

Comment:
I don't think this is a great function name as it just gets the first release found in a namespace.. it doesn't guarantee any specific one

Reply:
I've changed name to be **GetFirstReleaseInNamespace**
also I've modified description to meet the function